packages/kit/src/internal/esm.ts

```diff
@@ -21,17 +22,19 @@ export function directoryToURL (dir: string): URL {
  */
 export async function tryResolveModule (id: string, url: URL | URL[]): Promise<string | undefined>
 /** @deprecated pass URLs pointing at files */
-export async function tryResolveModule (id: string, url: string | string[]): Promise<string | undefined>
-export async function tryResolveModule (id: string, url: string | string[] | URL | URL[] = import.meta.url) {
-  try {
-    return await resolvePath(id, { url })
-  } catch {
-    // intentionally empty as this is a `try-` function
-  }
+export function tryResolveModule (id: string, url: string | string[]): Promise<string | undefined>
+export function tryResolveModule (id: string, url: string | string[] | URL | URL[] = import.meta.url) {
+  return Promise.resolve(resolveModulePath(id, {
```

Comment:
i know it is a breaking change to not to but extra promises add overhead in microbenchmarks